.github/workflows/build.yml

```diff
@@ -224,6 +226,26 @@ jobs:
         name: docker-image
         path: artifacts
 
+    - name: Generate sha256 checksums
+      if: github.event_name == 'release'
+      run: |
+        for filePath in $(find  artifacts/* -name '*.tar.gz' -or -name '*.zip')
```

Comment:
This will include our `win-x64`, `linux-x64`, `osx-x64`, and `portable` binaries